core/App.js

```diff
@@ -130,6 +130,7 @@ class App {
 	 * @return {Promise}
 	 */
 	start() {
+		logger.info(`Starting the server in port ${this.config.socket}.`)
```

Comment:
Unix socket may be used as this.config.socket. e.g. "/var/run/asmodee-werewolf.sock". The log message is a bit misleading sometimes. How about `Starting the server at ${this.config.socket}.`?